Resources/Other-Sources/sv.lproj/Localizable.strings

```diff
@@ -0,0 +1,46 @@
+"DifferentTemperatureFormat" = "Det är %@ %@ än %@%@.";
+"PrecipitationChanceFormat" = "%@ chans för %@ %@.";
+"SameTemperatureFormat" = "Det är %@ %@ som %@%@.";
+
+"Hotter" = "hetare";
+"Warmer" = "varmare";
+"Cooler" = "svalare";
+"Colder" = "kallare";
+/* The word temperature is added at the end since the word is needed to make the sentence understandable in this context */
+"Same" = "samma temperatur";
```

Comment:
Is there something we can do better in the code so that we don't need this workaround? cc @thoughtbot/ios.